src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,44 +1,93 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo,
+  selectTodo: (id: number) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, selectTodo }) => {
+  const {
+    id,
+    title,
+    completed,
+    userId,
+  } = selectedTodo;
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    const fetchUser = async () => {
+      try {
+        const selectedUser = await getUser(userId);
+
+        setUser(selectedUser);
+      } catch (error) {
+        throw new Error('Error while selecting user');
+      }
+    };
+
+    fetchUser();
+  }, [userId]);
+
+  const closeModal = () => {
+    selectTodo(0);
+  };
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {!user ? (
         <Loader />
       ) : (
-        <div className="modal-card">
+        <div className="modal-card" key={user.id}>
           <header className="modal-card-head">
             <div
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #
+              {id}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={closeModal}
             />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {title}
             </p>
 
             <p className="block" data-cy="modal-user">
-              {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+              <strong
+                className={
+                  classNames({
+                    'has-text-success': completed,
+                    'has-text-danger': !completed,
+                  })
+                }
```

Comment:
You can not use wrap this
```suggestion
                className={classNames({
                    'has-text-success': completed,
                    'has-text-danger': !completed,
                  })}
```